src/NuGet.Core/NuGet.Protocol/Utility/CachingUtility.cs

```diff
@@ -23,13 +25,12 @@ public static string ComputeHash(string value, bool addIdentifiableCharacters =
         {
             var trailing = value.Length > 32 ? value.Substring(value.Length - 32) : value;
             byte[] hash;
-            using (var sha = SHA1.Create())
+            using (var sha = SHA256.Create())
             {
                 hash = sha.ComputeHash(Encoding.UTF8.GetBytes(value));
             }
 
-            const string hex = "0123456789abcdef";
-            return hash.Aggregate(addIdentifiableCharacters ? "$" + trailing : string.Empty, (result, ch) => "" + hex[ch / 0x10] + hex[ch % 0x10] + result);
+            return ToHex(hash, HashLength);
```

Comment:
Why remove the `addIdentifiableCharacters` behavior?

Reply:
Thanks. Added it back.